template/before.sh.erb

```diff
@@ -54,5 +54,26 @@ if [[ ${WORKSPACE_FILE+x} ]]; then
   cp $WORKSPACE_FILE .
 fi
 
+JUPYTER_API="<%= context.jupyterlab_switch == "1" ? "lab" : "tree" %>"
+
+<%- if context.classroom == "1" -%>
+module purge
+module load <%= context.version %>
+
+if [[ -n "${OSC_CLASS_FILES+x}" ]]; then
```

Comment:
Could you add documentation above this block explaining how OSC_CLASS_FILES and OSC_CLASS_ID and OSC_CLASS_FILE all work, what their purpose is, etc.?

Reply:
and maybe an example value for each variable

Reply:
Gave descriptions and dropped `OSC_CLASS_FILE`.